tests/conftest.py

```diff
@@ -218,22 +223,22 @@ def test_filters_random_with_mask(filter_config: dict, test_filters: Filters, im
 @pytest.fixture
 def random_filters(test_filters_random_with_mask: Filters) -> Filters:
     """Process random with filters, for use in grains fixture."""
-    test_filters_random_with_mask.images["initial_align"] = test_filters_random_with_mask.align_rows(
+    test_filters_random_with_mask.images["initial_align"] = test_filters_random_with_mask.median_flatten(
         test_filters_random_with_mask.images["pixels"], mask=None
     )
     test_filters_random_with_mask.images["initial_tilt_removal"] = test_filters_random_with_mask.remove_tilt(
         test_filters_random_with_mask.images["initial_align"], mask=None
     )
-    test_filters_random_with_mask.images["masked_align"] = test_filters_random_with_mask.align_rows(
+    test_filters_random_with_mask.images["masked_align"] = test_filters_random_with_mask.median_flatten(
         test_filters_random_with_mask.images["initial_tilt_removal"], mask=test_filters_random_with_mask.images["mask"]
     )
     test_filters_random_with_mask.images["masked_tilt_removal"] = test_filters_random_with_mask.remove_tilt(
         test_filters_random_with_mask.images["masked_align"], mask=test_filters_random_with_mask.images["mask"]
     )
 
-    test_filters_random_with_mask.images["zero_averaged_background"] = test_filters_random_with_mask.average_background(
-        test_filters_random_with_mask.images["masked_tilt_removal"], mask=test_filters_random_with_mask.images["mask"]
-    )
+    # test_filters_random_with_mask.images["zero_averaged_background"] = test_filters_random_with_mask.average_background(
```

Comment:
Commented lines that are no longer used can be removed. I personally find it makes the diffs in PRs easier to read/review.

Reply:
I believe I fixed this earlier too